packages/app/src/store/specs-store.ts

```diff
@@ -12,16 +12,12 @@ export const useSpecStore = defineStore({
   state (): SpecState {
     return {
       activeSpec: undefined,
-      specFilter: undefined,
```

Comment:
Unused? I think a hold-over from before we used GraphQL and preferences to save the spec filter.